src/core/thread/mle.hpp

```diff
@@ -2024,7 +2030,10 @@ class Mle : public InstanceLocator, private NonCopyable
 #endif
 
     otMleCounters mCounters;
-
+#if OPENTHREAD_CONFIG_UPTIME_ENABLE
+    void     UpdateTimeCounter(DeviceRole aRole);
```

Comment:
Suggest moving the method decleration up next to other method declarations.
(we would need `#if OPENTHREAD_CONFIG_UPTIME_ENABLE` again but that is fine and better to have all methods declerations next to each other.).